src/helper-modules/ordnance-survey/exception/ordnance-survey.exception.test.ts

```diff
@@ -0,0 +1,28 @@
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+
+import { OrdnanceSurveyException } from './ordnance-survey.exception';
+
+describe('OrdnanceSurveyException', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const message = valueGenerator.string();
+
```

Comment:
An assertion to assert whether the exception status code is a number?

Reply:
This internal exception doesn't have http status code yet.